README.md

```diff
@@ -1,25 +1,233 @@
-The CZ Cell Census is an aggregation of all public single cell data available in [CELLxGENE Discover](https://cellxgene.cziscience.com/), published in API-accessible formats, including the [SOMA API](https://github.com/single-cell-data/).
 
-### Reuse
+Get started on using the Cell Census:
 
-The contents of this Github repository are freely available for reuse under the [MIT license](https://opensource.org/licenses/MIT). Data in the CZ Cell Census are available for re-use under the [CC-BY license](https://creativecommons.org/licenses/by/4.0/).
+- [Quick start](#Quick-start).
+- [Documentation](https://chanzuckerberg.github.io/cell-census/index.html). *Under development.*
+- [Python tutorials](https://github.com/chanzuckerberg/cell-census/tree/main/api/python/notebooks).
+- R tutorials. *Coming soon.*
 
-### For More Help
+## Motivation: Single-cell analysis at scale 
 
-Please file a issue on this repo, or contact us at <cellxgene@chanzuckerberg.com>
+The Cell Census aims to break the barrier of data fragmentation in the single-cell field by presenting a   cell-based slicing and querying access paradigm for all data at CZ CELLxGENE Discover.
 
-### Security
+The Cell Census is a data object publicly hosted online and a convenience API to open it. The object is built using the [SOMA](https://github.com/single-cell-data/SOMA) API and data model via its implementation [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA). As such, the Cell Census has all the data capabilities offered by TileDB-SOMA including:
 
-If you believe you have found a security issue, we would appreciate notification. Please send email to <security@chanzuckerberg.com>.
+- Cloud-based data storage and access.
+- Efficient access for larger-than-memory slices of data.
+- Data streaming for iterative/parallelizable  methods.
+- R and Python support.
+- Export to AnnData and Seurat.
 
-### Code of Conduct
+## Cell Census data releases
+
+Starting in  mid 2023, Cell Census long-term supported data builds will be released every 6 months and and will be publicly accessible for at least 5 years upon release. 
+
+In between long-term supported data build releases, weekly builds are released without any guarantee of permanence. 
+
+## Cell Census data organization
+
+The Cell Census follows a specific [data schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema.md). Briefly, the Cell Census is a collection of a variety of **SOMA** objects organized with the following hierarchy.
+
+<img src="./docs/cell_census_data_model.svg">
+
+
+## Quick start
+
+### Requirements
+
+The Cell Census requires a Linux or MacOS system with:
+
+- Python 3.7 to Python 3.10. Or R, supported versions TBD.
+- Recommended: >16 GB of memory.
+- Recommended: >5 Mbps internet connection. 
+- Recommended: for increased performance use the API through a AWS-EC2 instance from the region `us-west-2`. The Cell Census data builds are hosted in a AWS-S3 bucket in that region.
```

Comment:
the hyphen's in `AWS-S3` and `AWS-EC2` are incorrect (I believe). Should be `AWS S3...`, etc.